cocos/scripting/lua-bindings/manual/CCLuaEngine.cpp

```diff
@@ -37,8 +37,6 @@
 #include "base/CCDirector.h"
 #include "base/CCEventCustom.h"
 
-#pragma comment(lib,"lua51.lib")
```

Comment:
As i remember, it is used to link lua lib on windows.

Reply:
Yes, remove from sources, let user can specific which version of lua lib he want to link.

Reply:
Where to choose? The travis is blocked.

Reply:
lualib need add to the final lua test executable projects, I'll do later.

Reply:
And please modify the lua template too. Thanks.

Reply:
ok, I'll.